src/components/icons/ChevronRight.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { createIcon } from '@chakra-ui/react';
+
+export const ChevronRight = createIcon({
+  displayName: 'ChevronRight',
+  viewBox: '0 0 24 24',
+  path: (
+    <path
+      d="M7.59 7.41L12.17 12L7.59 16.59L9 18L15 12L9 6L7.59 7.41Z"
+      fill="#666666"
```

Comment:
I'm wondering if we need `currentColor` here for it future use in different cases.